### PR TITLE
[DOC-10075] Document new topology minimums for Advanced

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10075] Document new topology minimums for Advanced

Previews:
[src/current/cockroachcloud/advanced-cluster-management.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/advanced-cluster-management.html)
[src/current/cockroachcloud/ccloud-get-started.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/ccloud-get-started.html)
[src/current/cockroachcloud/cockroachdb-advanced-on-azure.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/cockroachdb-advanced-on-azure.html)
[src/current/cockroachcloud/create-an-advanced-cluster.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/create-an-advanced-cluster.html)
[src/current/cockroachcloud/plan-your-cluster-advanced.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/plan-your-cluster-advanced.html)
[src/current/cockroachcloud/production-checklist.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/production-checklist.html)
[src/current/cockroachcloud/quickstart-trial-cluster.md](https://deploy-preview-18880--cockroachdb-docs.netlify.app/docs/cockroachcloud/quickstart-trial-cluster.html)

### CODE REVIEW

File: src/current/cockroachcloud/plan-your-cluster-advanced.md [hunk 1]
```diff
@@ -40,15 +40,15 @@ You can configure a maximum of 9 regions per cluster through the Console. If you
 
 ### Cluster scaling
 
-When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For example, if you have a 3 node cluster with 2 vCPUs per node, consider scaling up to 8 vCPUs before adding a fourth node. For most production applications, we recommend at least 4 to 8 vCPUs per node.
+When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For example, if you have a 3 node cluster with 4 vCPUs per node, consider scaling up to 8 vCPUs before adding a fourth node. For most production applications, we recommend at minimum 8 vCPUs per node. New CockroachDB {{ site.data.products.cloud }} organizations can no longer scale a cluster down to fewer than 4 vCPUs per node.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:43]:
```suggestion
When scaling up your cluster, it is generally more effective to increase node size up to 16 vCPUs before adding more nodes. For example, if you have a 3 node cluster with 4 vCPUs per node, consider scaling up to 8 vCPUs before adding a fourth node. For most production applications, we recommend at minimum 8 vCPUs per node. New CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024 can no longer scale a cluster down to fewer than 4 vCPUs per node.
```

File: src/current/cockroachcloud/plan-your-cluster-advanced.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ You can configure a maximum of 9 regions per cluster through the Console. If you
...
 
-If you have changed the [replication factor](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/configure-replication-zones) for a cluster, you might not be able to remove nodes from the cluster. For example, suppose you have a 5-node cluster and you had previously changed the replication factor from its default value of 3 to 5. Now if you want to scale down the cluster to 3 nodes, the decommissioning nodes operation might fail. To successfully remove nodes from the cluster in this example, change the replication factor back to 3, and then remove the nodes.
+If you have changed the [replication factor](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/configure-replication-zones) for a cluster, you might not be able to remove nodes from the cluster. For example, suppose you have a 5-node cluster and you had previously changed the replication factor from its default value of 3 to 5. Now if attempt to scale down the cluster to 3 nodes, the decommission operation might fail. To successfully remove nodes from the cluster in this example, change the replication factor back to 3, and then remove the nodes.
```
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:47]:
```suggestion
If you have changed the [replication factor](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/configure-replication-zones) for a cluster, you might not be able to remove nodes from the cluster. For example, suppose you have a 5-node cluster and you had previously changed the replication factor from its default value of 3 to 5. Now if you attempt to scale down the cluster to 3 nodes, the decommission operation might fail. To successfully remove nodes from the cluster in this example, change the replication factor back to 3, and then remove the nodes.
```

File: src/current/cockroachcloud/advanced-cluster-management.md
Comment (jaiayu) [inline src/current/cockroachcloud/advanced-cluster-management.md:1]: This needs to be deleted. Apologies - I know we back and forth on this, but we're only removing the 2vcpu option for new orgs, not single node.

File: src/current/cockroachcloud/advanced-cluster-management.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ These sections show how to scale a {{ site.data.products.advanced }} cluster hor
 You can add or remove nodes from your cluster through the Console. See [Planning your cluster]({% link cockroachcloud/plan-your-cluster.md %}) for cluster requirements and recommendations before proceeding.
 
 {{site.data.alerts.callout_info}}
-You cannot scale a multi-node cluster down to a single-node cluster. If you need to scale down to a single-node cluster, [back up]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your cluster and [restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) it into a new single-node cluster.
+You cannot scale a multi-node cluster down to a single-node cluster. For new CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024, a cluster must have at least 3 nodes per region.
```
Comment (jaiayu) [inline src/current/cockroachcloud/advanced-cluster-management.md:37]: The second sentence here needs to be deleted. Apologies - I know we went back and forth on this, but we're not removing single node option for new orgs at this point. We are only removing the 2vcpu option for new orgs.

File: src/current/cockroachcloud/advanced-cluster-management.md [hunk 4]
```diff
@@ -100,7 +100,7 @@ You can add up to nine regions at a time through the Console. See [Planning your
 
     If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
 
-1. Select the desired new region and specify the number of nodes for it.
+1. Select the desired new region and specify the number of nodes for it. For new CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024, each region must have at least 3 nodes.
```
Comment (jaiayu) [inline src/current/cockroachcloud/advanced-cluster-management.md:103]: same as previous comment. Please delete the second sentence here.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 5]
```diff
@@ -118,13 +118,13 @@ Use the `ccloud cluster create` command to create a new CockroachDB {{ site.data
 ccloud cluster create dedicated
 ~~~
 
-This command creates a 1 node CockroachDB {{ site.data.products.dedicated }} cluster with 2 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
+This command creates a 3-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```
[the remark this answers is not in the document]
Comment (jaiayu) [reply src/current/cockroachcloud/ccloud-get-started.md:122]: Note: This command has not been updated yet with the default vCPUs set to 4. But we do plan to make this change. I'll circle back to confirm.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 6]
```diff
@@ -136,16 +136,16 @@ Success! Created cluster
...
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud cluster create dedicated blue-dog us-central1:2 us-west2:1 --cloud GCP --vcpus 2 --storage-gib 15
+ccloud cluster create dedicated blue-dog us-central1:8 us-west2:4 --cloud GCP --vcpus 4 --storage-gib 110
 ~~~
 </section>
 
 {{site.data.alerts.callout_info}}
-To set [resource limits](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits) for a CockroachDB {{ site.data.products.serverless }} cluster, or create a CockroachDB {{ site.data.products.dedicated }} cluster, you must [add a credit card](billing-management.html) to your organization.
+To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster, or create a CockroachDB {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
```
Comment (jaiayu) [inline src/current/cockroachcloud/ccloud-get-started.md:149]: This should be updated. 
- You can't set RUs anymore for Standard clusters. 
- For new orgs, you can't create any cluster without a credit card.

File: src/current/cockroachcloud/ccloud-get-started.md
Comment (jaiayu) [inline src/current/cockroachcloud/ccloud-get-started.md:1]: General comment on all of these - do we have confirmation that plan names are not changing in ccloud. I assumed the command may not change for continuity, but surprised to see the response also referencing the old plan names

File: src/current/cockroachcloud/cockroachdb-advanced-on-azure.md [hunk 7]
```diff
@@ -12,13 +12,6 @@ This page provides information about CockroachDB {{ site.data.products.dedicated
 
 CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
 
-### Editing and scaling
-
-- A cluster must have at minimum three nodes. A multi-region cluster must have at minimum three nodes per region. Single-node clusters are not supported.
```
Comment (jaiayu) [inline src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:17]: Since single-node will continue to be supported in GCP & AWS, maybe let's keep this.

File: src/current/cockroachcloud/advanced-cluster-management.md (hunk 3 above)
Comment (mdlinville) [inline src/current/cockroachcloud/advanced-cluster-management.md:37]:
```suggestion
You cannot scale a multi-node cluster down to a single-node cluster.
```

File: src/current/cockroachcloud/cockroachdb-advanced-on-azure.md [hunk 8]
```diff
@@ -12,13 +12,6 @@ This page provides information about CockroachDB {{ site.data.products.dedicated
...
-- After it is created, a cluster's storage can be increased in place, but cannot subsequently be decreased or removed.
```
Comment (jaiayu) [inline src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:18]: why is this deleted? Is it not true anymore?

File: src/current/cockroachcloud/advanced-cluster-management.md (hunk 4 above)
Comment (mdlinville) [inline src/current/cockroachcloud/advanced-cluster-management.md:103]:
```suggestion
1. Select the desired new region and specify the number of nodes for it.
```

File: src/current/cockroachcloud/ccloud-get-started.md
Comment (mdlinville) [reply src/current/cockroachcloud/ccloud-get-started.md:1]: `ccloud` has not been updated yet so the related docs have not been updated yet. This is getting merged into an integration branch for now.

File: src/current/cockroachcloud/create-an-advanced-cluster.md [hunk 9]
```diff
@@ -37,9 +37,8 @@ Select the region(s) and number of nodes for your cluster:
     A multi-region cluster requires at minimum three regions and can survive the loss of a single region. Refer to [Planning your cluster](plan-your-cluster-advanced.html?filters=advanced) for the configuration requirements and recommendations for CockroachDB {{ site.data.products.advanced }} clusters.
 
 1. Select the number of nodes:
-    - For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes also depends on your storage capacity and performance requirements. See [Example]({% link cockroachcloud/plan-your-cluster-advanced.md %}#example) for further guidance.
+    - For single-region production deployments, we require a minimum of 3 nodes. The number of nodes also depends on your storage capacity and performance requirements. See [Example]({% link cockroachcloud/plan-your-cluster-advanced.md %}#example) for further guidance. New CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024 can no longer create single-node clusters or scale a cluster to a single node. Single-node clusters are not suitable for production deployments.
```
Comment (jaiayu) [inline src/current/cockroachcloud/create-an-advanced-cluster.md:40]: Let's revert back to the original.

File: src/current/cockroachcloud/create-an-advanced-cluster.md [hunk 10]
```diff
@@ -37,9 +37,8 @@ Select the region(s) and number of nodes for your cluster:
...
     - For multi-region deployments, we require a minimum of 3 nodes per region. For best performance and stability, you should use the same number of nodes in each region.
-    - For single-region application development and testing, you may create a single-node cluster.
```
Comment (jaiayu) [inline src/current/cockroachcloud/create-an-advanced-cluster.md:42]: Let's not delete this anymore.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 11]
```diff
@@ -118,13 +119,13 @@ Use the `ccloud cluster create` command to create a new CockroachDB {{ site.data
 ccloud cluster create dedicated
 ~~~
 
-This command creates a 1 node CockroachDB {{ site.data.products.dedicated }} cluster with 2 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
+This command creates a 3-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:122]:
```suggestion
This command creates a 1-node CockroachDB {{ site.data.products.advanced }} cluster with 4 virtual CPUs (vCPUs) and 110 GiB of storage in the default cloud infrastructure provider (GCP) and the closest region for that provider. It will generate a cluster name. The CockroachDB version will be the latest stable version.
```

The `ccloud` commands have not yet been updated. I'm not sure if the tier names are being updated or we are going with small-s serverless and small-d dedicated.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 12]
```diff
@@ -118,13 +119,13 @@ Use the `ccloud cluster create` command to create a new CockroachDB {{ site.data
...
 
 You can set the cluster name, cloud infrastructure provider, region, number of nodes, and storage as command options. The following command is equivalent to the previous command that uses the default values.
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud cluster create dedicated blue-dog us-central1:1 --cloud GCP --vcpus 2 --storage-gib 15
+ccloud cluster create dedicated blue-dog us-central1:3 --cloud GCP --vcpus 4 --storage-gib 110
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:128]:
```suggestion
ccloud cluster create dedicated blue-dog us-central1:1 --cloud GCP --vcpus 4 --storage-gib 110
```

File: src/current/cockroachcloud/ccloud-get-started.md (hunk 6 above)
Comment (mdlinville) [reply src/current/cockroachcloud/ccloud-get-started.md:149]: I think for the free trial period, you do not yet need to add a credit card. Can you confirm?

File: src/current/cockroachcloud/plan-your-cluster-advanced.md [hunk 13]
```diff
@@ -16,14 +16,14 @@ Before making any changes to your cluster's configuration, review the requiremen
 
 ### Cluster configuration
 
-A single-node cluster is appropriate only for single-region application development and testing. For single-region production deployments, Cockroach Labs recommends a minimum of 3 nodes. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#example) for more information.
+For single-region production deployments, Cockroach Labs recommends a minimum of 3 nodes. For new CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024, 3 nodes is the minimum supported configuration. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#example) for more information.
```
Comment (jaiayu) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:19]: Let's revert to the original

File: src/current/cockroachcloud/plan-your-cluster-advanced.md [hunk 14]
```diff
@@ -16,14 +16,14 @@ Before making any changes to your cluster's configuration, review the requiremen
...
 
 Some of a CockroachDB {{ site.data.products.advanced }} cluster's provisioned memory capacity is used for system overhead factors such as filesystem cache and sidecars, so the full amount of memory may not be available to the cluster's workloads.
 
 CockroachDB {{ site.data.products.advanced }} clusters use three Availability Zones (AZs). For balanced data distribution and best performance, we recommend using a number of nodes that is a multiple of 3 (for example, 3, 6, or 9 nodes per region).
 
 {{site.data.alerts.callout_info}}
-You cannot scale a multi-node cluster down to a single-node cluster. If you need to scale down to a single-node cluster, [back up]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your cluster and [restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) it into a new single-node cluster.
+You cannot scale a multi-node cluster down to a single-node cluster.
```
Comment (jaiayu) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:26]: Let's revert back to the original

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 15]
```diff
@@ -136,16 +137,16 @@ Success! Created cluster
...
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud cluster create dedicated blue-dog us-central1:2 us-west2:1 --cloud GCP --vcpus 2 --storage-gib 15
+ccloud cluster create dedicated blue-dog us-central1:8 us-west2:4 --cloud GCP --vcpus 4 --storage-gib 110
 ~~~
 </section>
 
 {{site.data.alerts.callout_info}}
-To set resource limits for a CockroachDB {{ site.data.products.serverless }} cluster, or create a CockroachDB {{ site.data.products.dedicated }} cluster, you must [add a credit card](billing-management.html) to your organization.
+To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster, or create a CockroachDB {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:149]:
```suggestion
To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, or to create a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
```

File: src/current/cockroachcloud/production-checklist.md [hunk 16]
```diff
@@ -37,7 +35,7 @@ Make sure your cluster has sufficient storage, CPU, and memory to handle the wor
 `raw data (storage, in GB) * replication factor (3 by default) * remove 40% to account for compression (0.6) * headroom (1.5-2)`
 
 {{site.data.alerts.callout_info}}
-CockroachDB {{ site.data.products.advanced }} clusters can be created with a minimum of 2 vCPUs per node on AWS and GCP or 4 vCPUs per node on Azure.
+CockroachDB {{ site.data.products.advanced }} clusters can be created with a minimum of 4 vCPUs per node and a minimum of 3 nodes per region.
```
Comment (mdlinville) [inline src/current/cockroachcloud/production-checklist.md:38]:
```suggestion
CockroachDB {{ site.data.products.advanced }} clusters can be created with a minimum of 4 vCPUs per node.
```

File: src/current/cockroachcloud/create-an-advanced-cluster.md (hunk 9 above)
Comment (mdlinville) [inline src/current/cockroachcloud/create-an-advanced-cluster.md:40]:
```suggestion
    - For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes also depends on your storage capacity and performance requirements. See [Example]({% link cockroachcloud/plan-your-cluster-advanced.md %}#example) for further guidance.
```

File: src/current/cockroachcloud/plan-your-cluster-advanced.md (hunk 13 above)
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:19]:
```suggestion
For single-region production deployments, Cockroach Labs recommends a minimum of 3 nodes. The number of nodes you choose also affects your storage capacity and performance. See the [Example](#example) for more information.
```

File: src/current/cockroachcloud/cockroachdb-advanced-on-azure.md [hunk 17]
```diff
@@ -12,13 +12,6 @@ This page provides information about CockroachDB {{ site.data.products.dedicated
 
 CockroachDB {{ site.data.products.dedicated }} clusters on Azure have the following temporary limitations. To express interest or request more information about a given limitation, contact your Cockroach Labs account team. For more details, refer to the [FAQs](#faqs).
```
Comment (mdlinville) [inline src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:14]:
```suggestion
A cluster must have at minimum three nodes. A multi-region cluster must have at minimum three nodes per region. Single-node clusters are not supported on Azure.
```

File: src/current/cockroachcloud/cockroachdb-advanced-on-azure.md (hunk 8 above)
Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:18]: Because it's the same as for all clouds, and is noted in the Cluster Creation page.

File: src/current/cockroachcloud/production-checklist.md [hunk 18]
```diff
@@ -30,21 +28,21 @@ This page provides important recommendations for CockroachDB {{ site.data.produc
...
-When planning your deployment, it is important to carefully review and choose the [deployment options](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/choose-a-deployment-option) that best meet your scale, cost, security, and resiliency requirements.
+When planning your deployment, it is important to carefully review and choose the [deployment options]({% link {{site.current_cloud_version}}/choose-a-deployment-option.md %}) that best meet your scale, cost, security, and resiliency requirements.
 
 Make sure your cluster has sufficient storage, CPU, and memory to handle the workload. The general formula to calculate the storage requirement is as follows:
 
 `raw data (storage, in GB) * replication factor (3 by default) * remove 40% to account for compression (0.6) * headroom (1.5-2)`
 
 {{site.data.alerts.callout_info}}
-CockroachDB {{ site.data.products.advanced }} clusters can be created with a minimum of 2 vCPUs per node on AWS and GCP or 4 vCPUs per node on Azure.
+CockroachDB {{ site.data.products.advanced }} clusters can be created with a minimum of 4 vCPUs per node and a minimum of 3 nodes per region.
```
Comment (jaiayu) [inline src/current/cockroachcloud/production-checklist.md:38]: remove "and a minimum of 3 nodes per region".

File: src/current/cockroachcloud/create-an-advanced-cluster.md [hunk 19]
```diff
@@ -37,9 +37,8 @@ Select the region(s) and number of nodes for your cluster:
...
 
 1. Select the number of nodes:
-    - For single-region production deployments, we recommend a minimum of 3 nodes. The number of nodes also depends on your storage capacity and performance requirements. See [Example]({% link cockroachcloud/plan-your-cluster-advanced.md %}#example) for further guidance.
+    - For single-region production deployments, we require a minimum of 3 nodes. The number of nodes also depends on your storage capacity and performance requirements. See [Example]({% link cockroachcloud/plan-your-cluster-advanced.md %}#example) for further guidance. New CockroachDB {{ site.data.products.cloud }} organizations created after September 26, 2024 can no longer create single-node clusters or scale a cluster to a single node. Single-node clusters are not suitable for production deployments.
     - For multi-region deployments, we require a minimum of 3 nodes per region. For best performance and stability, you should use the same number of nodes in each region.
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-an-advanced-cluster.md:41]:
```suggestion
    - For multi-region deployments, we require a minimum of 3 nodes per region. For best performance and stability, you should use the same number of nodes in each region.
    - For single-region application development and testing, you may create a single-node cluster.
```

File: src/current/cockroachcloud/plan-your-cluster-advanced.md (hunk 14 above)
Comment (mdlinville) [inline src/current/cockroachcloud/plan-your-cluster-advanced.md:26]:
```suggestion
You cannot scale a multi-node cluster down to a single-node cluster. If you need to scale down to a single-node cluster, [back up]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) your cluster and [restore]({% link cockroachcloud/take-and-restore-customer-owned-backups.md %}) it into a new single-node cluster.
```

File: src/current/cockroachcloud/quickstart-trial-cluster.md [hunk 20]
```diff
@@ -26,7 +26,7 @@ For this tutorial, you will create a 3-node GCP cluster in the `us-west2` region
 
 1. For **VPC Peering**, use the default selection of **Use the default IP range**. Click **Next: Capacity**.
 
-1. On the **Capacity page**, select 2vCPU for **Compute per node** and a 35 GiB disk for **Storage per node**.
+1. On the **Capacity page**, select 4 vCPU for **Compute per node** and a 35 GiB disk for **Storage per node**.
 
     {{site.data.alerts.callout_info}}
     You can select up to 9 nodes, 4 vCPUs of compute, and 150 GiB of storage. The trial code will not apply to larger clusters.
```
Comment (jaiayu) [inline src/current/cockroachcloud/quickstart-trial-cluster.md:32]: Note: The "Dedicated" trial is also being deprecated completely. Anyone who has it can complete it, but orgs will no longer be able to access it. Can we remove references to the trial code.

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 21]
```diff
@@ -136,16 +137,16 @@ Success! Created cluster
...
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-ccloud cluster create dedicated blue-dog us-central1:2 us-west2:1 --cloud GCP --vcpus 2 --storage-gib 15
+ccloud cluster create dedicated blue-dog us-central1:8 us-west2:4 --cloud GCP --vcpus 4 --storage-gib 110
 ~~~
 </section>
 
 {{site.data.alerts.callout_info}}
-To set resource limits for a CockroachDB {{ site.data.products.serverless }} cluster, or create a CockroachDB {{ site.data.products.dedicated }} cluster, you must [add a credit card](billing-management.html) to your organization.
+To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, or to create a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.advanced }} cluster, you must [add a credit card](billing-management.html) to your organization.
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:149]:
```suggestion
To set [resource limits]({% link {{site.current_cloud_version}}/architecture/glossary.md %}#resource-limits) for a CockroachDB {{ site.data.products.basic }} cluster, scale a CockroachDB {{ site.data.products.standard }}'s nodes, or create a {{ site.data.products.advanced }} cluster, you must [add a payment method](billing-management.html) to your organization.
```

File: src/current/cockroachcloud/ccloud-get-started.md [hunk 22]
```diff
@@ -136,16 +137,16 @@ Success! Created cluster
...
 {{site.data.alerts.end}}
```
Comment (mdlinville) [inline src/current/cockroachcloud/ccloud-get-started.md:150]:
```suggestion
```
Removing this admonition from the `ccloud` command page since you can do all this stuff with free trial credits.

File: src/current/cockroachcloud/cockroachdb-advanced-on-azure.md (hunk 7 above)
Comment (jaiayu) [reply src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:17]: @mdlinville thoughts on this one?

Comment (mdlinville) [reply src/current/cockroachcloud/cockroachdb-advanced-on-azure.md:17]: This page is about Azure, and Azure does not allow single-node. clusters. I suppose I should put some portion of this back, but not the heading.

File: src/current/cockroachcloud/quickstart-trial-cluster.md (hunk 20 above)
Comment (mdlinville) [reply src/current/cockroachcloud/quickstart-trial-cluster.md:32]: This is being tracked separately (Mike is working on the trial docs) so i will not address it here.
